library/src/main/java/no/finntech/capturandro/Capturandro.java

```diff
@@ -70,7 +66,6 @@ public Capturandro build() {
     Capturandro(Builder builder) {
         super();
         this.context = builder.context;
-        applicationContext = context.getApplicationContext();
         this.capturandroCallback = builder.capturandroCallback;
```

Comment:
With rx, it feels a bit unnecessary to have a callback which contains the observables. Perhaps the importImageFromCamera/Gallery methods could just return a new Observable directly?